package.json

```diff
@@ -24,13 +31,14 @@
     "copy": "[ ! -d build/src/lua ] && mkdir -p build/src/lua ; cp -r src/lua/* build/src/lua",
     "build": "tsc && npm run-script copy",
     "build-tests": "tsc -p tests/tsconfig.json && npm run-script copy",
-    "test": "jest --coverage ./tests/*"
+    "test": "jest --coverage ./tests/*",
+    "check-licenses": "apache2-license-checker"
   },
   "dependencies": {
-    "@dojot/adminkafka": "0.0.1-alpha.9",
-    "@dojot/dojot-module": "0.0.1-beta.9",
-    "@dojot/dojot-module-logger": "0.0.1-alpha.5",
-    "@dojot/healthcheck": "^0.0.1-alpha",
+    "@dojot/adminkafka": "0.2.0",
+    "@dojot/dojot-module": "0.1.0",
```

Comment:
I think it should be 0.1.1. If I'm not wrong the version 0.1.0 is broken.

Reply:
Ok, will create o pr to dev too